package.json

```diff
@@ -76,11 +76,11 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@doc-tools/transform": "^3.3.2",
     "@gravity-ui/eslint-config": "^2.0.0",
-    "@gravity-ui/page-constructor": "^4.0.0",
+    "@gravity-ui/page-constructor": "^4.8.0",
     "@gravity-ui/prettier-config": "^1.0.1",
     "@gravity-ui/stylelint-config": "^1.0.0",
     "@gravity-ui/tsconfig": "^1.0.0",
-    "@gravity-ui/uikit": "^5.1.0",
+    "@gravity-ui/uikit": "^5.11.0",
```

Comment:
is UI are ok after update? We have overrides uikit styles for us design

Reply:
> is UI are ok after update? We have overrides uikit styles for us design

It looks okey